expression/builtin_math.go

```diff
@@ -552,6 +552,49 @@ func builtinSign(args []types.Datum, ctx context.Context) (d types.Datum, err er
 	return d, nil
 }
 
+type sqrtFunctionClass struct {
```

Comment:
Please add a definition of builtinFunctionClass and builtinSprtSig at the beginning of this file.